backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -1,4 +1,5 @@
 using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
```

Comment:
Don't use Newtonsoft use the built in JsonSerializer that you can get the configuration from DI.

Reply:
That serializer doesn't support populateObject which is critical to my use case. I can probably find some reflection code on stack overflow that does the same thing, but that seems worse then using a published library.

https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to#populate-existing-objects

Reply:
Is this to support `metadata`?

Reply:
yes.

Reply:
Ugh, this isn't ideal on any level other than to quickly implement today.  Using the DAL layer (entities) for serialization is a big mistake that bites later on.  Sure it helps with RAD, but boy does it cause problems.  It's why we have models in the API for all serialization.  Due to the timing, we may as well let this slide, but we'll have to add it to tech-debt.